server/app/views/components/ToastMessage.java

```diff
@@ -44,7 +44,8 @@ public static ToastMessage alert(String message) {
     return new ToastMessage(message, ToastType.ALERT);
   }
 
-  public static ToastMessage error(String message) {
+  public static ToastMessage error(String message, boolean isLocalized) {
```

Comment:
We shouldn't have an `isLocalized` param here; if a language isn't localized yet, we will default to English, so "Error: " will be prepended until we get that TOAST_ERROR_MSG_OUTLINE string translated later, anyways. We don't want to remember to change this later on.

Reply:
The reason for the `isLocalized` param isn't for languages that aren't localized yet, it's for all the error toasts on the admin side which aren't being localized at all (aren't using the Play MessageProvider).  This is the case for all but two of the error toasts.

Reply:
Got it - sorry I didn't pay closer attention. In that case, can we make two methods, `ToastMessage.errorNonLocalized()` and `ToastMessage.error()` ? I think that will be easier to understand than a raw boolean passed in as the second param.

My second comment still stands for `ToastMessage.error()`, especially as we may get more toast errors on the applicant side in the future.

Reply:
I agree that having two methods will be clearer.  Thanks for the good idea!  I'll go ahead and add that.